app/models/db_connection.rb

```diff
@@ -263,4 +255,16 @@ def indexes(key:, database_name:, table_name:)
     end
   end
 
+  private
+
+  def default_database_name
+
+    initial_db || DEFAULT_DATABASE_NAME[db_type]
```

Comment:
This line returns `nil` if `initial_db` is `nil` and the user is using an SQLite-DB, which would let the connection fail. On another note: Since SQLite can only be used locally, make sure the users databases are properly separated if they ever will be able to create databases themselves...

Reply:
indeed, this is an open issue. and if we want to support sqlite, we would have to upload the db-file to the server (or have access to it through other options...). I created an [Issue](https://github.com/lebalz/db-sql/issues/10#issue-447669469) to address this later.